subsys/bluetooth/audio/has.c

```diff
@@ -114,7 +114,7 @@ static struct has_client {
 		bool pending_active_index;
 		bool pending_cp;
 		uint8_t preset_changed_index_next;
-	} ntf_bonded;
+	} ntf;
```

Comment:
This might be a nitpick, but considering that you have the union member just above also named ntf, did you consider renaming either for diffrentiation? I.e. ntf_paired?

Reply:
Or perhaps just `pending_ntf`?